docs/src/pages/system/basics/basics.md

```diff
@@ -1,338 +1,215 @@
 
-In order to use the `Box` component, you first need to create it.
-To start with, add a `spacing` and `palette` function to the style argument.
+## How do we solve this?
 
-```jsx
-import styled from 'styled-components';
-import { spacing, palette } from '@material-ui/system';
+In order to solve these issues, we need to have a simple way of pulling & wiring the correct design tokens for specific CSS properties, and adding them directly on the react element where we want the styles to be applied, by a prop, that can be easily discoverable.
 
-const Box = styled.div`
-  ${spacing}${palette}
-`;
+The `sx` prop, as part of the system is the solution we see for solving these problems. In the example above, you may see how it can be used on the MUI components.
 
-export default Box;
-```
+The property behaves as a superset of CSS that offers mapping values from the theme directly, by pulling specific values depending on the CSS property used. In addition to this, it allows a simple way of defining responsive values, that corresponds to the breakpoints values defined in the theme.
 
-This Box component now supports new [spacing props](/system/spacing/#api) and [color props](/system/palette/#api).
-For example, you can provide a padding prop: `p` and a color prop: `color`.
+With it you can build easily your custom visual components, like `Card`, `Badge`, `Chip` that could accept the props & behave exactlly as your design system specifies.
 
-```jsx
-<Box p="1rem" color="grey">
-  Give me some space!
-</Box>
-```
-
-The component can be styled providing any valid CSS values.
+In the next sections, we will dive deeper into all features of the `sx` prop.
 
-### Theming
-
-But most of the time, you want to rely on a theme's values to increase the UI consistency.
-It's preferable to have a predetermined set of padding and color values.
-Import the theme provider of your styling solution.
+## Installation
 
 ```jsx
-import * as React from 'react';
-import { ThemeProvider } from 'styled-components';
-
-const theme = {
-  spacing: 4,
-  palette: {
-    primary: '#007bff',
-  },
-};
+// with npm
+npm install @material-ui/system
 
-export default function App() {
-  return <ThemeProvider theme={theme}>{/* children */}</ThemeProvider>;
-}
+// with yarn
+yarn add @material-ui/system
 ```
 
-Now, you can provide a spacing multiplier value:
+## The `sx` prop
 
-```jsx
-<Box p={1}>4px</Box>
-<Box p={2}>8px</Box>
-<Box p={-1}>-4px</Box>
-```
+The `sx` prop is a superset of CSS, that let's you add any CSS to the underlaying component, while mapping values to different theme keys. This should help you to keep your application look consistent. The prop is available in all `@material-ui/core` components.
```

Comment:
Duplicate of https://github.com/mui-org/material-ui/pull/23294/files#diff-9cfcff806f98bc7c0817a102aa630285ea875a8161eb89c2ad992e2659e62fa6R33

Reply:
Good call, fixed the section. Please take another look